src/__tests__/lib/exampleData.js

```diff
@@ -142,6 +149,24 @@ export const makeStream = (args: { name?: string, description?: string } = {}):
   });
 };
 
+const randGroupId: () => number = makeUniqueRandInt('group IDs', 1000);
+export const makeGroup = (args: { name?: string, description?: string } = {}): UserGroup => {
+  const name = args.name ?? randString();
+  const description = args.description ?? `This is a group for discussing ${randString()}`;
+  const members = [];
+
+  for (let i = 0; i < randInt(10) + 1; i++) {
+    members.push(randInt(1000));
+  }
```

Comment:
As written, this means the same user could be present in a group twice. I suspect that might violate some expectations.

Additionally, `i < randInt(10) + 1` is being executed on every pass through the loop; this means the output of `randInt` will be different every time, and the average number of loops will be much less than 5.